### ISSUE TITLE
JavaFX canvas apply font faces only after reload the web-page

### ISSUE THREAD

**Issue by bekasov** [JPro-one/JPro-tickets#162]
Hello, there is issue with supporting fonts on JavaFX canvas - canvas draw fonts correctly only if reload the web-page.

Gradle: 8.1.1
JVM:          11.0.19 (Eclipse Adoptium 11.0.19+7)
OS:           Linux 6.3.5-2-MANJARO amd64
FX: 17.0.6
Browser: Firefox 114.0.1
Start app cmd: gradle jproRun

Test app:
```java
public class HelloJPro extends Application {
    @Override
    public void start(Stage primaryStage) {
		BorderPane root = new BorderPane();
        Font sansSerif = Font.font("SansSerif", 20);
        Font roboto = Font.font("Roboto", 20);

        System.out.println(Font.getFontNames());
        System.out.println(Font.getFamilies());
        System.out.println(Font.getDefault());
        Canvas canvas = new Canvas(500, 300);
        final GraphicsContext graphicsContext2D = canvas.getGraphicsContext2D();
        graphicsContext2D.setFont(roboto);
        graphicsContext2D.fillText("Canvas: Some text Roboto", 0, 30);
        graphicsContext2D.setFont(sansSerif);
        graphicsContext2D.fillText("Canvas: Some text SansSerif", 0, 100);
        System.out.println(graphicsContext2D.getFont());
        final Text someText1 = new Text("Text: Some text Roboto");
        someText1.setFont(roboto);
        final Text someText2 = new Text("Text: Some text SansSerif");
        someText2.setFont(sansSerif);
        VBox hBox = new VBox(canvas, someText1, someText2);
        root.setLeft(hBox);
        primaryStage.setScene(new Scene(root, 800, 600));
        primaryStage.show();
    }

    /**
     * Application entry point.
     *
     * @param args the command line arguments
     */
    public static void main(String[] args) {
        launch(args);
    }
}
```

result just after auto-open the page:
![image](https://github.com/JPro-one/JPro-tickets/assets/7324734/d15cdba2-bcc3-4533-a0df-e27a8822a7ad)

after reload the web-page:
![image](https://github.com/JPro-one/JPro-tickets/assets/7324734/515a1796-51e6-4bd4-b442-298a5a778039)

**Comment by FlorianKirmaier** [comment]
Thank you for the bug report!

After refreshing, how do you get back to the "non-working state"?
By using incognito? Or by restarting the JPro Server?

**Comment by bekasov** [comment]
> Thank you for the bug report!
> 
> After refreshing, how do you get back to the "non-working state"? By using incognito? Or by restarting the JPro Server?

This is an unstable behavior.
With each page reloading, we catch a different results.
Sometimes, it renders correctly, sometimes it renders with wrong font.

**Comment by FlorianKirmaier** [comment]
Fixed in the upcoming version 2023.2.2 , testable with 2023.2.2-SNAPSHOT
